features/let-const.yml

```
@@ -0,0 +1,8 @@
+name: Let and const
+description: The `let` and `const` declarations define block-scoped variables.
+spec: https://tc39.es/ecma262/multipage/ecmascript-language-statements-and-declarations.html#sec-let-and-const-declarations
```

I see we've come around on https://github.com/web-platform-dx/web-features/pull/330 😂 

```suggestion
spec: https://tc39.es/ecma262/multipage/ecmascript-language-statements-and-declarations.html#sec-let-and-const-declarations
caniuse:
  - let
  - const
```